components/Badge/Badge.css

```diff
@@ -0,0 +1,67 @@
+@import url(/Asimov-UI/style.css);
+@import url(/Asimov-UI/components/Avatar/Avatar.css);
+.badge-wrapper {
+  width: fit-content;
+  position: relative;
+
+  display: flex;
+
+  align-items: center;
+  justify-content: center;
+}
+
+.badge {
+  position: absolute;
+
+  border-radius: 50%;
+  font-weight: bold;
+  display: flex;
+
+  align-items: center;
+  justify-content: center;
+}
+
+.badge-top {
+  top: -11px;
+  right: -13px;
+}
+
+.badge-status {
+  right: 1%;
+  bottom: 0.5%;
+  border: 3px solid var(--bg-color);
+  height: calc(var(--avatar-height) / 4);
+  width: calc(var(--avatar-width) / 4);
+}
+
+.badge-s {
+  width: 20px;
+  height: 20px;
+
+  font-size: 0.9rem;
+}
+
+.badge-red {
+  background-color: red;
+  color: white;
+}
+
+.badge-green {
+  background-color: green;
+  color: white;
+}
+
+.badge-yellow {
+  background-color: rgb(245, 210, 12);
```

Comment:
derive color from variable

Reply:
Hey Koushik thanks for the review!
 that is the only place that color will be used hence I left it that way